pkg/queue/cluster_queue_strict_fifo_test.go

```diff
@@ -38,7 +39,10 @@ func TestFIFOClusterQueue(t *testing.T) {
 		Spec: kueue.ClusterQueueSpec{
```

Comment:
I actually preferred putting `&kueue.ClusterQueue` in a new line.

Reply:
Change back to new line